app/addons/documents/index-results/stores.js

```diff
@@ -179,11 +180,27 @@ function (FauxtonAPI, ActionTypes, HeaderActionTypes, Documents, MangoHelper) {
     },
 
     getResults: function () {
+      var hasEditableAndDeleteableDoc;
```

Comment:
hasEditableAndDelet///e///ableDoc --> hasEditableAndDeletableDoc,

and maybe this should be a logic thing? if(isEditableDoc && isDeletableDoc)then...